BFit/Controller/FollowingViewController.swift

```diff
@@ -30,11 +31,12 @@ class FollowingViewController: UIViewController, UITableViewDataSource, UITableV
     }
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        numFollowing.text = "\(mockData.list.count) Following"
         return mockData.list.count
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath) as UITableViewCell
+        let cell = UITableViewCell.init()
```

Comment:
Good call, if our app needs to scale we can research this more but for the current scale this is totally fine.